src/request.rs

```diff
@@ -41,8 +45,26 @@ pub trait KvRequest: Sync + Send + 'static + Sized {
     }
 
     fn response_stream(
+        self,
+        pd_client: Arc<impl PdClient>,
+    ) -> BoxStream<'static, Result<Self::RpcResponse>> {
+        let backoff_state = BackoffState::new(
+            REGION_RETRY_BASE_DELAY_MS,
+            REGION_RETRY_MAX_DELAY_MS,
+            REGION_RETRY_ATTEMPTS,
+        );
+
+        let backoff = Arc::new(NoJitterBackoff {
```

Comment:
It looks a bit strange that we need to create that backoff even if there isn't any error...

Reply:
If you think a backoff is too large to copy, I think the `Copy` and `Clone` trait on `Backoff` is unnecessary.

Reply:
@sticnarf Hi, I fix to remove `Arc` and wrap `impl` Backoff` with Option. So it could be assigned to `None::<NoJitterBackoff>` for the first time. Please help review the update. Thanks.